### DISCUSSION TITLE
If you want to include a new dataset in TDC, let us know here ⬇️

### DISCUSSION THREAD

**Discussion by kexinhuang12345** [mims-harvard/TDC#6] [New Datasets]
Hi! 

Drop any pointers for a therapeutics dataset that you find may be appropriate for TDC here! 

We would love to discuss it with you and also work with you to include it in TDC!

**Comment by abearab** [comment]
We have recently released a set of multi-omics datasets in the context of understanding a drug's behavior in different biological backgrounds. 

https://www.biorxiv.org/content/10.1101/2022.12.14.518457v2

Mainly, our three large-scale functional genomics studies can be useful for downstream machine learning tasks. In [this tweet](https://x.com/abe_bio/status/1702581336475566430?s=20) I tried to share some of my thoughts in this regard. I would love to discuss possibilities of including these type of datasets into TDC.

Please let me know what you think :)

**Reply by kexinhuang12345** [reply]
Hi Abolfazl, thanks for reaching out - looks pretty interesting and relevant! It would be great to chat - just sent an email to abarbiology@gmail.com for scheduling!
